uber/templates/macros.html

```diff
@@ -162,7 +162,7 @@
         });
     </script>
 
-    {% if is_prereg_dealer %}
+    {% if is_dealer and not group %}
```

Comment:
The check for collect_full_address is already done OUTSIDE the macro. If we are calling the macro, then we're gonna print the all the address fields. The check is not useful.